libsignal-service/src/proto.rs

```diff
@@ -65,3 +67,17 @@ impl WebSocketResponseMessage {
         }
     }
 }
+
+impl SyncMessage {
+    pub fn with_padding() -> Self {
```

Comment:
@gferon Shouldn't we make those stronger types, such that it's impossible to transmit without padding?

Reply:
Yes, that was a ninja trick for the time being.